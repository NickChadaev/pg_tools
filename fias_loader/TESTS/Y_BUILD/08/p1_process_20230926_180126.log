2023-09-26 18:01:26 home/n.chadaev@abrr.local
127.0.0.1 5432 unsi_test_08 postgres /home/n.chadaev@abrr.local/Y_BUILD/08/stage_gar_c_08.csv 223-09-25
--------------------------------------------------------
[18:01:26] ... Калмыкия Респ, id_region: 9, Execute: /home/n.chadaev@abrr.local/Y_BUILD/08/stage_gar_c_08.csv, process_id:5456
[18:01:26] ... Start process
[18:01:26] ...  -- Version
[18:01:26] ...  -- Очистка от данных
[18:01:26] ...  -- Set UNLOGGED
[18:01:26] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/AS_APARTMENT_TYPES_20230925_dec7f8ad-4238-4484-bf99-113a09c762f5.XML
[18:01:26] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/AS_ADDR_OBJ_TYPES_20230925_4f714402-bdf0-4b03-8e2f-cda2704fb403.XML
[18:01:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/AS_ROOM_TYPES_20230925_6fedd268-7191-415b-be0e-8a5a110f7099.XML
[18:01:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/AS_OPERATION_TYPES_20230925_d65578ac-f625-4e50-ac1b-06ee9ac4de7a.XML
[18:01:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/AS_PARAM_TYPES_20230925_72393625-9278-4105-b37d-79dc852810ae.XML
[18:01:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/AS_HOUSE_TYPES_20230925_1bdc09e7-b8f0-40d5-80f3-cf003785714b.XML
[18:01:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/AS_ADDHOUSE_TYPES_20230925_cf9caa45-fb3f-40d3-8ddf-770f536de4ed.XML
[18:01:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/AS_OBJECT_LEVELS_20230925_8ba71762-ad23-4052-b0b8-a659eac50bbb.XML
[18:01:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/AS_NORMATIVE_DOCS_TYPES_20230925_acf1cb3d-a66a-4e4b-b233-cc9c4032d98a.XML
[18:01:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/AS_NORMATIVE_DOCS_KINDS_20230925_a8475f50-3b47-4725-9f46-b1bd99ea576c.XML
[18:01:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/08/AS_REESTR_OBJECTS_20230925_0b5b682e-f6a7-4c01-a937-8c45424150f3.XML
[18:02:57] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/08/AS_ADDR_OBJ_20230925_7f8c441d-8e84-4b48-91b3-022f2f2cbcc8.XML
[18:03:01] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/08/AS_HOUSES_20230925_c76efca7-ab6c-4e69-a5cd-972600e8d89c.XML
[18:04:58] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/08/AS_STEADS_20230925_85623ccf-5399-490e-8e2a-001ff0166aab.XML
[18:04:59] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/08/AS_APARTMENTS_20230925_7a7026ee-27e2-4345-beab-29d397ae9b6f.XML
[18:05:42] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/08/AS_ROOMS_20230925_86ba3a05-237e-4a4e-a3c9-585e766e1568.XML
[18:05:42] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/08/AS_CARPLACES_20230925_fcb7cf87-963f-44f2-8405-f8bef1cabc5c.XML
[18:05:42] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/08/AS_ADM_HIERARCHY_20230925_e153a3c3-200e-4489-8869-3bcc45534c59.XML
[18:07:33] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/08/AS_MUN_HIERARCHY_20230925_fa4ed3f4-b615-4f8c-a980-16e6e8ebd596.XML
[18:09:19] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/08/AS_CHANGE_HISTORY_20230925_6e0304b3-9ba9-4ab8-b9ab-71aa0326dd74.XML
[18:15:10] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/08/AS_ADDR_OBJ_DIVISION_20230925_0c8a1db5-6537-46d7-b9cd-4f256918b81d.XML
[18:15:10] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/08/AS_ADDR_OBJ_PARAMS_20230925_334f0927-adde-4bdb-9bd8-da4b015f5b60.XML
[18:15:54] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/08/AS_STEADS_PARAMS_20230925_1650b804-8b93-4b88-a35a-7e0e52f02759.XML
[18:16:01] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/08/AS_HOUSES_PARAMS_20230925_88094f4f-8c35-429d-a39b-fdc7b93082ef.XML
[18:27:45] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/08/AS_APARTMENTS_PARAMS_20230925_fa20d66e-870d-4892-9193-1d661d40cbfc.XML
[18:29:19] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/08/AS_ROOMS_PARAMS_20230925_e1401112-d7db-4769-b7d8-ce379aa0f8d7.XML
[18:29:19] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/08/AS_CARPLACES_PARAMS_20230925_b189da13-4426-48e6-aa27-1bfe2325ca29.XML
[18:29:19] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/08/AS_NORMATIVE_DOCS_20230925_b64f8201-b475-487c-819d-beb926edc830.XML
[18:29:49] ...  -- Data finish
[18:29:49] ...  -- Set LOGGED
[18:29:54] ... Stop process
--------------------------------------------------------
