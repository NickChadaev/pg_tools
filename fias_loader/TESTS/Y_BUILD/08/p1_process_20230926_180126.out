 save_gar_version 
------------------
               70
(1 row)

CALL
CALL
 set_gar_dt_create 
-------------------
                70
(1 row)

CALL
gar_fias.as_apartment_type: 13
gar_fias.as_addr_obj_type: 421
gar_fias.as_room_type: 3
gar_fias.as_operation_type: 27
gar_fias.as_param_type: 20
gar_fias.as_house_type: 14
gar_fias.as_add_house_type: 4
gar_fias.as_object_level: 17
gar_fias.as_norm_docs_types: 25
gar_fias.as_norm_docs_kinds: 4
gar_fias.as_reestr_objects: 135115
gar_fias.as_addr_obj: 5010
gar_fias.as_houses: 146512
gar_fias.as_steads: 976
gar_fias.as_apartments: 54750
gar_fias.as_rooms: 60
gar_fias.as_carplaces: 0
gar_fias.as_adm_hierarchy: 135880
gar_fias.as_mun_hierarchy: 135784
gar_fias.as_change_history: 473776
gar_fias.as_addr_obj_division: 8
gar_fias.as_addr_obj_params: 57808
gar_fias.as_steads_params: 9799
gar_fias.as_houses_params: 975753
gar_fias.as_apartments_params: 133535
gar_fias.as_rooms_params: 64
gar_fias.as_carplaces_params: 0
gar_fias.as_normative_docs: 41205
